tests/php/ORM/Search/SearchContextTest.php

```diff
@@ -59,12 +59,18 @@ public function testSearchableFieldsDefaultsToSummaryFields()
         $summaryFields = $obj->summaryFields();
         $expected = [];
         foreach ($summaryFields as $field => $label) {
+            if ($field === 'Image.CMSThumbnail'
+                || $field === 'Image.BackLinks.Count') {
+                continue;
+            }
```

Comment:
Why are we skipping these for this test?

Reply:
`CMSThumbnail ` and `Count` are helper methods and don't have fields in the database. See my comment above.